rubrics/tool/src/main/frontend/webcomponents/sakai-item-delete.js

```diff
@@ -52,7 +52,7 @@ export class SakaiItemDelete extends SakaiElement {
   render() {
 
     return html`
-      <span role="button" aria-haspopup="true" aria-expanded="${this.popoverOpen}" aria-controls="delete_${this.type}_${this.item.id}" tabindex="0" title="${tr("remove", [this.item.title])}" class="delete fa fa-times" @click="${this.deleteItem}"></span>
+      <a href="#" role="button" aria-haspopup="true" aria-expanded="${this.popoverOpen}" aria-controls="delete_${this.type}_${this.item.id}" tabindex="0" title="${tr("remove", [this.item.title])}" class="delete fa fa-times linkStyle" @click="${this.deleteItem}"></a>
```

Comment:
I am wondering if this will have the undesired side effect of scrolling to the top of the page. Normally, in the absence of `event.preventDefault()` in the bound function, an `<a href="#">` will result in scrolling to the top of the page.